src/Events/Editor/Full_Site/Hooks.php

```diff
@@ -98,6 +130,32 @@ public function filter_include_templates( $query_result, $query, $template_type
 		return array_merge( $query_result, $single_events_template, $events_archive_template );
 	}
 
+	/**
+	 * Fetch our Block Template by ID.
+	 *
+	 * @since TBD
+	 *
+	 * @param null|WP_Block_Template $block_template The filtered template.
+	 * @param string                 $id             The block template ID.
+	 * @param string                 $template_type  The template type.
+	 *
+	 * @return null|WP_Block_Template
+	 */
+	public function filter_include_template_by_id( $block_template, $id, $template_type ) {
+		if ( ! is_null( $block_template ) ) {
+			return $block_template;
+		}
```

Comment:
```suggestion
		}

```
let's fix the spacing here